proto/p4/v1/p4runtime.proto

```diff
@@ -500,7 +500,10 @@ message StreamMessageResponse {
     PacketIn packet = 2;
     DigestList digest = 3;
     IdleTimeoutNotification idle_timeout_notification = 4;
-    google.protobuf.Any other = 5;
+    .google.protobuf.Any other = 5;
+    // Used by the server to asynchronously report errors which occur when
+    // processing StreamMessageRequest messages.
+    .google.rpc.Status error_status = 6;
   }
```

Comment:
I think we should have a separate message, maybe called "PacketOutError" that contains a gooogle.rpc.Status, instead of directly having the status here.  I think that would be much clearer.  Right now the comment/message reads as if this error can be used for any error wrt. StreamMessageRequest, but that is not the case.  E.g., MasterArbitrationUpdate have their own error handling.

Reply:
My assumption was that it could be used for other types of notification (e.g. `DigestListAck`), and I gave an example in the spec, but I agree that there is some redundancy for master arbitration.